### DISCUSSION TITLE
Add DiskZIP

### DISCUSSION THREAD

**Discussion by installaware** [IviriusCommunity/Rebound#15] [Ideas]
You currently feature a third party app.

Another worthy addition would be InstallAware's DiskZIP https://www.installaware.com/diskzip-security-privacy-performance.htm .

From InstallAware Software, makers of https://github.com/installaware/AGPT

https://www.installaware.com/diskzip.exe

**Comment by IviriusMain** [comment]
This app does not comply with the design principles of Rebound 11, is there a WinUI 3 version of DiskZIP available?
